### PR TITLE
multiple smart punctuation fixes

### PR DESCRIPTION
**Pull request by rwxrob**
Still needs strike but otherwise quotes are golden in headers and elsewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,44 +15,52 @@ from `vim-pandoc/vim-pandoc`.
...
-to `.vimrc`, and then run `:PluginInstall`.
+**Vundle** users should add:
 
-For those who need it, a tarball is available from
-[here](https://github.com/vim-pandoc/vim-pandoc-syntax/archive/master.zip).
+```vim
+Plugin 'vim-pandoc/vim-pandoc-syntax'
+```
+
+Run `:PluginInstall` after making the changes from within a `vim` session, or you can just run `vim +':PlugInstall' +':q!' +':q!'` from the command line.
```
Comment (alerque) [inline README.md]:
```suggestion
Run `:PluginInstall` after making the changes and reloading your rc file from within a `vim` session, or you can just run `vim +PlugInstall +UpdateRemotePlugins +all +qa` from the command line.
```

File: README.md [hunk 2]
```diff
@@ -16,44 +15,52 @@ from `vim-pandoc/vim-pandoc`.
...
 * Configurable (see `:help pandoc-syntax-configuration` for an overview of the
   options).
 
 ## Screenshots
 
-![img1](http://i.imgur.com/UKXbG2V.png)
-![img2](http://i.imgur.com/z8FpxRP.png)
-![img3](http://i.imgur.com/ziNjQiE.png)
-![img4](http://i.imgur.com/UKoOxzP.png)
+![img1](doc/UKXbG2V.png)
```
Comment (alerque) [inline README.md]: Can you please revert these changes and drop the image blobs from the commit(s)? For now I think I'd rather leave these out-of-band. One you put screenshots in a plugin they follow the repository around literally forever, and if you update them people start having multiple versions of the images in their repositories. Given that I check out my plugins on all sorts of machines, that starts adding up space and bandwidth for very little gain.

**Comment by alerque** [comment]
Thanks for taking the time to contribute this @rwxrob. The bulk of the vim stuff looks alright at first pass (although I haven't tested it in action yet), but I have a couple requests in regard to the Readme changes. If you can, I'd like to see the unrelated changes about plugin managers etc. in a separate commit (in case something goes haywire and we have to `git bisect` or `revert`). Are you familiar with rebasing? The easiest tool for that job by the way is `git revise -c` to split at existing commit by hunks.

**Comment by rwxrob** [comment]
Hope it's okay to just close this request and open a new one #313 with only the changes you suggest. I'm not (yet) acquainted with `rebase` and `revise` but till focus on it for next time.

I understand now your reason for not including images. That makes a lot of sense.

There is another bug with relation to `:::` divs that I would like to work on later as well. Perhaps I'll submit the report first so we can track it. Thanks again.

**Pull request closed without merging**